java8/src/test/java/io/cucumber/java8/TypeDefinitionsStepDefinitions.java

```diff
@@ -18,8 +19,8 @@
 
     public TypeDefinitionsStepDefinitions() {
         Given("docstring, defined by lambda",
-            (StringBuilder builder) -> assertThat(builder.getClass(), equalTo(StringBuilder.class)));
-        DocStringType("doc", (String docString) -> new StringBuilder(docString));
+            (DocString builder) -> assertThat(builder.getClass(), equalTo(DocString.class)));
+        DocStringType("doc", DocString::create);
```

Comment:
I don't understand why this was changed. Can you explain?

Reply:
This was me trying to make it work before making io/cucumber/java8/LambdaGlue.java:409 generic:
 default <T> void DocStringType(String contentType, DocStringDefinitionBody<T> body) {} 
 I reverted this